### PR TITLE
[Merged by Bors] - PLAT-566: added auth service sender == delegateTo check and test

### PR DESCRIPTION
**Pull request by sexyhamster**
**- What I did**
- added check for sender == delegateTo
- added test for C5194

### CODE REVIEW

File: ledger-core/virtual/integration/delegationtoken_test.go [hunk 1]
```diff
@@ -193,6 +196,18 @@ func TestDelegationToken_IsMessageFromVirtualLegitimate(t *testing.T) {
...
+		{
+			name:       "Fail if wrong delegate",
+			testRailID: "C5194",
+			approverVE: veSetFixed,
+			currentVE:  veSetFixed,
+			errorMessages: []string{
+				"token DelegateTo and sender are different",
+				"illegitimate msg",
+			},
+			errorSeverity:  0,
```
Comment (MikhailZhukov) [inline ledger-core/virtual/integration/delegationtoken_test.go:208]: test case required severity=remote breach

Comment (sexyhamster) [reply ledger-core/virtual/integration/delegationtoken_test.go]: fixed

**Pull request closed without merging**